README.md

```diff
@@ -144,6 +151,11 @@ sudo apt-get update
 sudo apt-get install oracle-java8-installer
 ```
 
+### Installing grunt
+```
+npm install -g grunt-cli
```

Comment:
This *might* be undesirable if we ever had different versions of grunt in different packages in the monorepo. Then we'd have to use a local `node_modules/.bin` grunt. But, I don't think we do, so 🤷‍♂

Reply:
Agreed. We've recommended this way for the whole duration of the project, so I'm codifying it so newcomers aren't blocked. Let's discuss installation in individual repos in an issue.

Reply:
https://github.com/mozilla/fxa/issues/1156